AdblockPlusSafari/AdblockPlusController.m

```diff
@@ -64,7 +66,7 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
 {
   if ([keyPath isEqualToString:NSStringFromSelector(@selector(enabled))]) {
     self.blockingEnablingSwitch.on = self.adblockPlus.enabled;
-    [self.tableView reloadSections:[NSIndexSet indexSetWithIndex:1] withRowAnimation:UITableViewRowAnimationNone];
+    [self.tableView reloadSections:[NSIndexSet indexSetWithIndexesInRange:NSMakeRange(0, 2)] withRowAnimation:UITableViewRowAnimationNone];
```

Comment:
An unfortunate binding of momentary design to hardcoded numbers. Someone will forget to update this when/if the design changes around. Like when a new section is inserted between. If such section should NOT be disabled on ABP toggle, then Range will be a wrong tool altogether. Any idea of how to make this programmatically safe?